prometheus-export/src/main/scala/StreamingQuerySource.scala

```diff
@@ -33,8 +33,11 @@ object StreamingQuerySource {
     value.split("\\[").last.split("\\]").head
   }
 
+  /** extract the source name from the description for example
+    * "KafkaV2[Subscribe[mpathic-event]]" becomes "KafkaV2"
+    */
   def extractSourceName(value: String): String = {
-    value.split("\\[", 1).head
+    value.split("\\[").head
```

Comment:
I think `2` would work here. Not a blocker, I just realized how confusing that second argument is.